internal/service/advancedclustertpf/resource.go

```diff
@@ -136,15 +133,15 @@ func (r *rs) Update(ctx context.Context, req resource.UpdateRequest, resp *resou
 		return
 	}
 
-	stateUsingLegacy := usingLegacySchema(ctx, state.ReplicationSpecs, diags)
```

Comment:
aligning terminology of `ShardingConfig` as mentioned in our docs and sdk v2 implementation